src/main/java/com/hackerrank/weather/model/Genre.java

```diff
@@ -0,0 +1,21 @@
+package com.hackerrank.weather.model;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "genres")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Genre {
+    @Id
+//    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @JsonAlias("name")
```

Comment:
Jackson would create JSON fields following the java field name, so no need for that @JsonAlias.